helm-charts/mock-classifier/templates/tests/test-connection.yaml

```diff
@@ -0,0 +1,15 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: "{{ include "mock-classifier.fullname" . }}-test-connection"
+  labels:
+    {{- include "mock-classifier.labels" . | nindent 4 }}
+  annotations:
+    "helm.sh/hook": test
+spec:
+  containers:
+    - name: wget
+      image: busybox
+      command: ['wget']
+      args: ['{{ include "mock-classifier.fullname" . }}:{{ .Values.service.port }}']
```

Comment:
I don't know how `busybox` works but will this be continuously pinging our service or will it be on a regular but spaced interval?

Reply:
If I remember correctly then `busybox` is just set of shell utilities used commonly with small distros like Alpine linux. So I'm guessing this test is something like readiness probe.

Reply:
Correct, busy box is a collection of bash utilities, this part was pre-generated, and wget is used as a download command, essentially, I think this is trying to test whether we can download resources from the server, I think this is very similar to the readiness probe essentially, we can just replicate the readiness probe here maybe, there is an issue here to further refine the tests, would be a good discussion https://github.com/buerokratt/Infrastructure/issues/24